cmd/ferretdb/main.go

```diff
@@ -92,8 +92,8 @@ var cli struct {
 	Test struct {
 		RecordsDir string `default:"" help:"Testing: directory for record files."`
 
-		DisableFilterPushdown bool `default:"false" help:"Experimental: disable filter pushdown."`
-		EnableOplog           bool `default:"false" help:"Experimental: enable capped collections, tailable cursors and OpLog." hidden:""`
+		DisablePushdown bool `default:"false" help:"Experimental: disable all pushdown."`
```

Comment:
I think it should be "all pushdowns" or (probably better) just "pushdown", without "all"

Reply:
That comment was about the help text, not field name